src/Type/Definition/InterfaceType.php

```diff
@@ -104,12 +103,14 @@ public function getInterfaces(): array
                 );
             }
 
-            /** @var array<int, InterfaceType> $interfaces */
-            $interfaces = $interfaces === null
-                ? []
-                : array_map([Schema::class, 'resolveType'], $interfaces);
-
-            $this->interfaces = $interfaces;
+            $this->interfaces = [];
```

Comment:
I guess it's not a problem now but shouldn't we do atomic swap instead?

Reply:
I only see that leading to more code would even guess that the intermediary variable costs performance.

Reply:
The intermediary variable should be removed by php optimalizations.

i'm used to doing that when working with multiple threads. It is not a thing in php yet but might be in the future (maybe even with Fibers in 8.1?)

Reply:
It is done.